src/library/scala/collection/View.scala

```diff
@@ -163,6 +163,33 @@ object View extends IterableFactory[View] {
     def iterator = partition.underlying.iterator.filter(x => partition.p(x) == cond)
   }
 
+  /** A view that splits an underlying collection into two views */
+  @SerialVersionUID(3L)
```

Comment:
BTW this class is not `Serializable`, so annotating it with `@SerialVersionUID` produces a warning (that’s also true for some other classes -- that you didn’t write -- in this file)

Reply:
Yes Indeed! I ll remove them 👍